Sources/Ollama/Client.swift

```diff
@@ -25,6 +25,9 @@ open class Client {
 
     /// The underlying client session.
     internal(set) public var session: URLSession
+	
+	/// A list of tools to include in chat requests
+	var tools: [Chat.Tool] = []
```

Comment:
This parameter appears to be internal as well, not sure the reason.

Reply:
An oversite. Fixed now.